ports/espressif/boards/m5stack_stamp_s3/mpconfigboard.h

```diff
@@ -0,0 +1,29 @@
+// This file is part of the CircuitPython project: https://circuitpython.org
+//
+// SPDX-FileCopyrightText: 2025 Stella Schwankl for Adafruit Industries
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2025 Stella Schwankl
```
I only add the "for..." bit because I'm paid by them. I don't think you need to.

Reply:
Got it!